### PR TITLE
Add check before accessing lesson ID. (Get Next Lesson)

### PR DESCRIPTION
**Pull request by kjohnson**

### CODE REVIEW

File: includes/class.llms.lesson.php [hunk 1]
```diff
@@ -348,7 +348,12 @@ public function get_next_lesson() {
 			if ($sections) {
 				$newsection = new LLMS_Section( $sections[0]->ID );
 				$lessons = $newsection->get_children_lessons();
-				return $lessons[0]->ID;
+
+				if( $lessons ){
```
Comment (thomasplevy) [inline includes/class.llms.lesson.php]: please add a space after the `if` so it reads `if (`

**Comment by thomasplevy** [comment]
@kjohnson thanks for the fix here, can you fix the the one line as per our coding standards documentation before we merge it in. Thanks so much for understanding my knit-pickyness here and for taking the time to actually resolve the issue rather than just yelling at me about it!

**Comment by thomasplevy** [comment]
@kjohnson also credit where credit's due -- do you want a mention and a link to somewhere in our changelog on the next release? I'll do something along the lines of 

"Thanks to <name>(link) for contributing a fix in relation to <issue>"

_if you'd like_

**Comment by kjohnson** [comment]
@thomasplevy a mention would be appreciated, thanks :)

Good for you for enforcing coding standards!

Also, you may want to squash my extra commits.

**Comment by thomasplevy** [comment]
@kjohnson I (usually) squash. Thanks for understanding.

I'll be working on getting a small release out by the end of the week (probably) and will test and included this with that release.

Thanks so much,

**Comment by thomasplevy** [comment]
@kjohnson merged manually into develop for testing before release. Expect this in LifterLMS 2.7.5 in the next day or so. Thanks so much!

**Pull request closed without merging**